hypercube/Dockerfile

```diff
@@ -29,9 +29,7 @@ RUN --mount=type=cache,id=hypercube-apk,sharing=locked,from=cache,target=/var/ca
     addgroup nginx jwt && \
     cleanup.sh
 
-ENV \
-    HYPERCUBE_FCREPO_URL=fcrepo/fcrepo/rest \
-    HYPERCUBE_LOG_LEVEL=info
+ENV HYPERCUBE_LOG_LEVEL=info
```

Comment:
Still seems to be in use: https://github.com/Islandora-Devops/isle-buildkit/blob/main/hypercube/rootfs/etc/confd/templates/config.yaml.tmpl#L7

Reply:
If this isn't needed, it should be removed from the `config.yaml.tmpl` file as well.

Reply:
Apologies for delay. Removed now

Reply:
lol, turns out it was supposed to be there. https://islandora.slack.com/archives/CM6F4C4VA/p1668431519198479